frontend/components/organisms/labels/LabelCreationForm.vue

```diff
@@ -18,19 +18,18 @@
           type="error"
           dismissible
         >
-          The label could not be created.
-          You cannot use same label name or shortcut key.
+          {{ $t('errors.labelCannotCreate') }}
         </v-alert>
         <v-text-field
           v-model="labelName"
           :rules="labelNameRules"
```

Comment:
Maybe forget to change.

The same thing for `colorRules`:

```
...
:rules="colorRules"
...
```

Reply:
Addressed in https://github.com/doccano/doccano/pull/976/commits/aa2679edd15b629335e74a10781123373b0e7880